### PR TITLE
Add --bucket-bandwidth flag to `mc admin replicate update`

### PR DESCRIPTION
**Pull request by poornas**
to allow default bucket bandwidth to be set in site replication

## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license] (https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## How to test this PR?
with minio [PR](https://github.com/minio/minio/pull/18062) 

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

File: cmd/admin-replicate-update.go [hunk 1]
```diff
@@ -48,7 +48,10 @@ var adminReplicateUpdateFlags = []cli.Flag{
 		Usage:  "enable synchronous replication for this target, valid values are ['enable', 'disable'].",
 		Value:  "disable",
 		Hidden: true, // deprecated Jul 2023
-
+	},
+	cli.StringFlag{
+		Name:  "bucket-bandwidth",
+		Usage: "Set default bandwidth limit for bucket in bits per second (K,B,G,T for metric and Ki,Bi,Gi,Ti for IEC units)",
```
Comment (shtripat) [inline cmd/admin-replicate-update.go:54]: Please add help entries for these flags like `bucket-bandwidth`, `mode`, `sync` etc

**Review by shtripat — COMMENTED** [review]
I have a site replication set across two local MinIO instances as below
```
./mc admin replicate info m1
SiteReplication enabled for:

Deployment ID                        | Site Name       | Endpoint                                       | Sync | Bandwidth 
                                     |                 |                                                |      | Per Bucket
d4a37885-92bf-490f-8b13-8758b14b4252 | m1              | http://localhost:9000                          |      | N/A       
1c8d3dee-4dde-4b0a-91c4-a8a0ba5397e4 | m2              | http://localhost:9010                          |      | N/A       
```

But below two commands display the help only
1. `./mc admin replicate update --deployment-id d4a37885-92bf-490f-8b13-8758b14b4252 --endpoint http://localhost:9010`
2. `./mc admin replicate update --deployment-id d4a37885-92bf-490f-8b13-8758b14b4252 --endpoint http://localhost:9010 --bucket-bandwidth 2000`

Anything wrong here?

**Review by harshavardhana — APPROVED** [review]

**Comment by harshavardhana** [comment]
PTAL at the conflicts @poornas

**Review by harshavardhana — APPROVED** [review]

**Review by shtripat — APPROVED** [review]
lgtm

**Pull request merged**